packages/lambda-powertools-logger/index.d.ts

```diff
@@ -0,0 +1,51 @@
+import CorrelationIds from "@dazn/lambda-powertools-correlation-ids";
+export = dazn__lambda_powertools_logger;
+declare enum LogLevels {
+  DEBUG = 20,
+  INFO = 30,
+  WARN = 40,
+  ERROR = 50,
+}
+
+declare class dazn__lambda_powertools_logger {
+  correlationIds: CorrelationIds;
+  originalLevel: string;
+  level: string;
+  constructor(argument?: { correlationIds?: CorrelationIds; level?: string });
+
+  appendError<T>(params: T, err: Error): T & Error;
+
+  debug(message: string, params?: any): void;
+
+  enableDebug(): () => void;
+
+  error(message: string, err?: Error): void;
+  error(message: string, params?: any, err?: Error): void;
+
+  info(message: string, params?: any): void;
+
+  isEnabled(level: number): Boolean;
+
+  log(levelName: string, message: string, params?: any): void;
+
+  resetLevel(): void;
+
+  warn(message: string, err?: Error): void;
+  warn(message: string, params?: any, err?: Error): void;
+
+  static debug(message: string, params?: any): void;
+
+  static enableDebug(): () => void;
```

Comment:
Is this a typo?
```suggestion
  static enableDebug(): void;
```

Reply:
`enableDebug () {
    this.level = 'DEBUG'
    return () => this.resetLevel()
  }`
The code is returning function to reset the log level and not return anything.

Reply:
You're right 👍 sorry just double checked this.